Makefile

```diff
@@ -131,13 +131,13 @@ TF_CFLAGS_aarch32	=	-target arm-arm-none-eabi -march=armv8-a
 TF_CFLAGS_aarch64	=	-target aarch64-arm-none-eabi -march=armv8-a
 else ifneq ($(findstring clang,$(notdir $(CC))),)
 TF_CFLAGS_aarch32	=	-target armv8a-none-eabi
-TF_CFLAGS_aarch64	=	-target aarch64-elf
+TF_CFLAGS_aarch64	=	-target aarch64-elf -mstrict-align
```

Comment:
Just realized, shouldn't this be in the else clause on line 137?  You are adding -mstrict-align for the clang configuration.